web/webkit/src/main/scala/net/liftweb/http/LiftServlet.scala

```diff
@@ -114,14 +114,21 @@ class LiftServlet extends Loggable {
     }
 
     if (reqOrg.request.suspendResumeSupport_?) {
-      runFunction(liftResponse => {
-        // do the actual write on a separate thread
-        Schedule.schedule(() => {
-          reqOrg.request.resume(reqOrg, liftResponse)
-        }, 0.seconds)
-      })
+      try {
+        reqOrg.request.suspend(cometTimeout)
+      } finally {
+        // FIXME This should be removed from the finally block in the next major
+        // FIXME Lift version; it is currently here to ensure any code that
+        // FIXME depends on a response handler running even if requests suspension
+        // FIXME fails continues to work as-is during the 3.x series.
+        runFunction(liftResponse => {
```

Comment:
Should we log a warning in the even we catch an exception here advising people of the impending change of behavior? Actually logging a `debug` or `warn` in general seems like a reasonable idea in the even suspension isn't working, but I also don't know how frequently that actually occurs.